cmd/webserver.go

```diff
@@ -0,0 +1,202 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+	"time"
+
+	webscan "github.com/Method-Security/webscan/generated/go"
+	webserver "github.com/Method-Security/webscan/internal/webserver"
+	webservertype "github.com/Method-Security/webscan/internal/webserver/type"
+	"github.com/spf13/cobra"
+)
+
+// InitWebServerCommand initializes the probe command for the webscan CLI. This command is used to perform a web probe against
+// targets to identify the existence of web servers.
+func (a *WebScan) InitWebServerCommand() {
+	webServerCmd := &cobra.Command{
+		Use:   "webserver",
+		Short: "Perform webserver analysis",
+		Long:  `Perform webserver analysis`,
+	}
+
+	probeCmd := &cobra.Command{
+		Use:   "probe",
+		Short: "Perform a web probe against targets to identify existence of web servers",
+		Long:  `Perform a web probe against targets to identify existence of web servers`,
+		Run: func(cmd *cobra.Command, args []string) {
+			targets, err := cmd.Flags().GetString("targets")
+			if err != nil {
+				errorMessage := err.Error()
```

Comment:
While we're in here, would be amazing to flip these over to the new error handling